pyflask/manageNeuroconv/manage_neuroconv.py

```diff
@@ -24,3 +24,26 @@ def get_all_interface_info() -> dict:
                 }
 
     return interface_info
+
+
+def get_combined_schema(interfaces):
```

Comment:
```suggestion
def get_combined_schema(interfaces: Optional[List[str]]) -> dict:
```

Again some more to learn about annotation typing; not only can you annotate the expected return type, but with `:` you can indicate the expected form of the inputs.

In this particular case, `Optional` means it can be the thing in `[...]` or `None` (more precisely, `Opional[...] = Union[..., None]`)

And I would actually not think to allow `None` as an input here - if the user has not selected any interfaces, then we simply should not proceed.

So I think this could become `interfaces: List[str]`

Note you will need to add an import for `from typing import List`

Reply:
Thank you for the explanation!